atom/browser/atom_browser_client.cc

```diff
@@ -322,6 +323,11 @@ void AtomBrowserClient::AppendExtraCommandLineSwitches(
         web_contents, command_line);
     SessionPreferences::AppendExtraCommandLineSwitches(
         web_contents->GetBrowserContext(), command_line);
+
+    auto context_id = atom::api::WebContents::GetIDForContents(
+      web_contents);
+    command_line->AppendSwitchASCII(switches::kContextId,
+      base::IntToString(context_id));
```

Comment:
@MarshallOfSound here we should use `base::Int64ToString`, which should fix half of the problem in https://github.com/electron/electron/issues/8205.

`atom::api::WebContents::GetIDForContents` returns `int64_t`, I'm not very sure why the compiler didn't warn about this typing thing.

Reply:
@sethlu good find

Reply:
🤔 Prolly `auto context_id` was interpreted as `int64_t`!?